main/src/main/java/com/sedmelluq/discord/lavaplayer/source/twitch/TwitchStreamAudioSourceManager.java

```diff
@@ -47,6 +48,16 @@ public TwitchStreamAudioSourceManager() {
     httpInterfaceManager = HttpClientTools.createDefaultThreadLocalManager();
   }
 
+  /**
+   * Create an instance.
+   * @param clientId The Twitch client id for your application.
+   */
+  public TwitchStreamAudioSourceManager(String clientId)
+  {
+      this();
+      twitchClientId = clientId;
+  }
```

Comment:
i feel like having a no-arg constructor that *only* calls the arg constructor might be a cleaner solution, which can be done by just having `TwitchStreamAudioSourceManager()` -> `this(CLIENT_ID)` and having the one with client-id set both the `httpInterfaceManager` and the used client id.

Reply:
My apologies; I'm having issues getting this to compile in IntelliJ. The suggested changes make a lot of sense, and I added them in as well; thank you!